packages/shared/src/components/ShareBar.tsx

```diff
@@ -42,53 +53,155 @@ export default function ShareBar({ post }: { post: Post }): ReactElement {
+
+      <div className="flex flex-wrap gap-2">
         <SimpleTooltip content="Copy link">
-          <ShareButton
+          <Button
+            className="group flex-col mb-2 font-normal text-center text-salt-90 hover:bg-transparent border-0 share btn-tertiary typo-caption1"
             onClick={trackAndCopyLink}
             pressed={copying}
-            icon={<CopyIcon />}
-            className="btn-tertiary-avocado"
-          />
+            textPosition="justify-between"
+            icon={
+              <div className="flex justify-center items-center mb-2 group-hover:bg-theme-hover rounded-xl">
+                <CopyIcon secondary className="w-7 h-7" />
+              </div>
+            }
+          >
+            Copy link
+          </Button>
         </SimpleTooltip>
+
         <SimpleTooltip content="Share on WhatsApp">
-          <ColorfulShareButton
+          <Button
+            className="group flex-col mb-2 font-normal text-center text-salt-90 hover:bg-transparent border-0 share btn-tertiary typo-caption1"
             tag="a"
             href={getWhatsappShareLink(href)}
             target="_blank"
             rel="noopener"
             onClick={() => onClick(ShareProvider.WhatsApp)}
-            icon={<WhatsappIcon secondary />}
-            className="btn-tertiary"
-          />
+            textPosition="justify-between"
+            icon={
+              <div className="flex justify-center items-center mb-2 group-hover:bg-theme-hover rounded-xl">
+                <WhatsappIcon secondary className="w-7 h-7 text-white" />
+              </div>
+            }
+          >
+            WhatsApp
+          </Button>
         </SimpleTooltip>
-        <SimpleTooltip content="Share on Twitter">
-          <ColorfulShareButton
+
+        <SimpleTooltip content="Share on Facebook">
+          <Button
+            className="group flex-col mb-2 font-normal text-center text-salt-90 hover:bg-transparent border-0 share btn-tertiary typo-caption1"
             tag="a"
-            href={getTwitterShareLink(href, post.title)}
+            href={getFacebookShareLink(href)}
             target="_blank"
             rel="noopener"
-            onClick={() => onClick(ShareProvider.Twitter)}
-            icon={<TwitterIcon secondary />}
-            className="btn-tertiary"
-          />
+            onClick={() => onClick(ShareProvider.Facebook)}
+            textPosition="justify-start"
+            icon={
+              <div className="flex justify-center items-center mb-2 group-hover:bg-theme-hover rounded-xl">
+                <FacebookIcon secondary className="w-7 h-7 text-white" />
+              </div>
+            }
+          >
+            Facebook
+          </Button>
         </SimpleTooltip>
-        <SimpleTooltip content="Share on Facebook">
-          <ColorfulShareButton
+
+        <SimpleTooltip content="Share on Twitter">
+          <Button
```

Comment:
Reference on color, also suggestion maybe just create a generic classList of functional component?